server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/plan/node/write/InsertMultiTabletsNode.java

```diff
@@ -119,11 +128,13 @@ public void addInsertTabletNode(InsertTabletNode node, Integer parentIndex) {
 
   @Override
   public void setSearchIndex(long index) {
+    searchIndex = index;
```

Comment:
Why need this?

Reply:
For InsertMultiTabletsNode, InsertRowsNode and InsertRowsOfOneDeviceNode, return value of getSearchIndex method should be same with its sub nodes.